webrtc.html

```diff
@@ -3378,36 +3378,38 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
 
           <dt> sequence&lt;RTCRtpContributingSource&gt;
             getContributingSources ()  </dt>
-          <dd> <p>Returns a RTCRtpContributingSource for each unique
+          <dd> <p>Returns an <code><a>RTCRtpContributingSource</a></code> for each unique
               CSRC or SSRC received by this RTCRtpReceiver in the last 10
-            seconds. </p>
+            seconds.</p>
           </dd>
               
 	</dl>
 
-    <p> The RTCRtpContributingSource objects contains information information
-    about a given contribution source, including the time the most recent time a
+    <p> The <dfn>RTCRtpContributingSource</dfn> objects contain information
+    about a given contributing source, including the time the most recent time a
     packet was received from the source. The browser MUST keep information from
-    RTP packets received in the previous second. When a RTP packet is received,
-    the entries are updated for the SSRC of the RTP packet and for any CSRC
-    contained in it. </p>
+    RTP packets received in the previous 10 seconds.  Each time an RTP packet is received, 
+    the <code><a>RTCRtpContributingSource</a></code> objects are updated.  If the RTP packet 
+    contains CSRCs, then the <code><a>RTCRtpContributingSource</a></code> objects corresponding 
+    to those CSRCs are updated.  If the RTP packet contains no CSRCs, then the <code><a>RTCRtpContributingSource</a></code> 
+    object corresponding to the SSRC is updated.</p>
 
 	 <dl class="idl" title="interface RTCRtpContributingSource">
 
-	   <dt> readonly attribute DOMHighResTimeStamp timestamp</dt> <dd> Time of
-	   reception of the most recent RTP packet containing the contributing
-	   source. </dd>
+	   <dt> readonly attribute DOMHighResTimeStamp timestamp</dt> 
+	   <dd> <p>The timestamp of type DOMHighResTimeStamp [HIGHRES-TIME], indicating the time of 
+	   reception of the most recent RTP packet containing the source.  
+	   The timestamp corresponds to the local clock.</p> </dd>
```

Comment:
Actually that's not what I interpreted it as, nor is it what was my "alternative C" on the list - that was the time of *reception* of the packet, not the receiver's interpretation of the time it was sent.